stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/chat/ChatFragment.kt

```diff
@@ -61,6 +62,13 @@ class ChatFragment : Fragment() {
     private var _binding: FragmentChatBinding? = null
     private val binding get() = _binding!!
 
+    /**
+     * [DownloadPermissionHandler] instance tasked with requesting [Manifest.permission.WRITE_EXTERNAL_STORAGE]
+     * permission. By default will ask for permission and if it is granted will download the file.
+     */
+    private val downloadHandler: DownloadPermissionHandler = DownloadPermissionHandler()
```

Comment:
can we make this a lazy property

Reply:
Conflicts with the activity lifecycle stuff